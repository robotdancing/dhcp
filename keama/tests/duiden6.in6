# EN server duid config

# authoritative is mandatory
authoritative;

# EN server duid declaration
server-duid en 2495 "enterprise-specific-identifier-1234";
