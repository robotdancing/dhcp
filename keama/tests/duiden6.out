{
  # EN server duid config
  # authoritative is mandatory
  "Dhcp6": {
    # EN server duid declaration
    "server-id": {
      "type": "EN",
      "enterprise-id": 2495,
      "identifier": "enterprise-specific-identifier-1234"
    }
  }
}
